services/backend/db/pg_hba.conf

```diff
@@ -0,0 +1,100 @@
+# TYPE DATABASE USER CIDR-ADDRESS  METHOD
+host  all  all 0.0.0.0/0 trust
```

Comment:
Since we don't use local host in our setup due to the docker network, this will just allow all traffic to pass. If this was anything else we could set it to the static IP for the agent I believe but didn't seem like an issue since it's a lab